packages/claim-cli/src/utils/get-private-key.ts

```diff
@@ -0,0 +1,95 @@
+import { select } from '@inquirer/prompts';
+import * as crypto from '@liskhq/lisk-cryptography';
+import { Mnemonic, HDNodeWallet, Wallet } from 'ethers';
+import { remove0x } from './prefix';
+import { getInput, getPassword } from './get-prompts';
+
+enum SecretType {
+	MNEMONIC,
+	PRIVATE_KEY,
+	JSON,
+}
+
+const getSecretType = (wallet: string) =>
+	select({
+		message: `Secret Type for ${wallet}`,
+		choices: [
+			{ name: 'Mnemonic', value: SecretType.MNEMONIC },
+			{ name: 'Private Key', value: SecretType.PRIVATE_KEY },
+		],
+	});
+
+export async function getLSKPrivateKeyFromMnemonic(): Promise<Buffer> {
+	const mnemonic = await getPassword({ message: 'Your Mnemonic' });
+	if (!Mnemonic.isValidMnemonic(mnemonic)) {
+		console.log('Invalid Mnemonic, please check again.');
+		return process.exit(1);
```

Comment:
shouldn't this be `throw new Error`? process.exit will stop any cleanup if it exist in the future